guides/customization/vnc.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using VNC with Coder
+description: Learn how to use a VNC with Coder
+---
+
+This guide will show you how to set up a Virtual Network Computing (VNC) system
+in Coder. Coder does not have a specific set of VNC providers it supports. Coder
+will render the VNC, as long as it is installed on the image used to create the
```

Comment:
Let me know if there are VNCs that might cause trouble, or that we should call out here.